Python/instrumentation.c

```diff
@@ -1000,7 +1000,7 @@ instrumentation_cross_checks(PyInterpreterState *interp, PyCodeObject *code)
     return monitors_equals(code->_co_monitoring->active_monitors, expected);
 }
 
-static inline
+static
```

Comment:
My previously comment was mostly intended to be that `static int` should be on one line